plugins/lang-expression/src/main/java/org/elasticsearch/script/expression/ExpressionPlugin.java

```diff
@@ -19,24 +19,22 @@
 
 package org.elasticsearch.script.expression;
 
-import org.elasticsearch.ElasticsearchException;
-import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.plugins.Plugin;
+import org.elasticsearch.script.ScriptModule;
 
-import java.io.IOException;
+public class ExpressionPlugin extends Plugin {
 
-/**
- * Exception used to wrap exceptions occuring while running expressions.
- */
-public class ExpressionScriptExecutionException extends ElasticsearchException {
-    public ExpressionScriptExecutionException(String msg, Throwable cause) {
-        super(msg, cause);
+    @Override
+    public String name() {
+        return "lang-expression";
```

Comment:
Not related at all but I think we should change names to "script-expression" (same for other "lang-" plugins).
We should do that in another PR for 3.0